src/drawers/areaDrawer.ts

```diff
@@ -16,7 +16,7 @@ export module Drawers {
     }
 
     public selectionForIndex(index: number) {
-      return this.renderArea().select(this.selector());
```

Comment:
The newer version is faster because selection() is cached. The `d3.select` only puts the wrapper around the item. The old version is doing a DOM lookup for the `this.selector()` which is slower